.github/workflows/start-minor-release.yml

```
@@ -0,0 +1,57 @@
+---
+name: "Start minor release"
+
+on: # yamllint disable-line rule:truthy
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "The Determined minor version to release. E.g. 0.38.0. This will create a new release branch and make commits on main."
+        required: true
+
+jobs:
+  start-minor-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: "Validate version"
+        shell: bash {0}
+        run: |
+          grep -E -o '[0-9]+\.[0-9]+\.0' <(printf "%s" '${{ github.event.inputs.version }}')
```

No love for herestrings?
```suggestion
          grep -E -o '[0-9]+\.[0-9]+\.0' <<<'${{ github.event.inputs.version }}'
```

---

Well I'll be! :D